### PR TITLE
SEO-SpellingCorrection

### PR DESCRIPTION
**Pull request by TameemAnsari-SF4702**
Change the meaningless word into meaningful word. Using the keywords in hidden.

### CODE REVIEW

File: MAUI/TextInputLayout/Assistive-Labels.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ inputLayout.Content = new Entry();
 
 The visibility of the helper text can be disabled by setting the [ShowHelperText](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfTextInputLayout.html#Syncfusion_Maui_Core_SfTextInputLayout_ShowHelperText) property to false. By default, it is set to true.
 
-![helper label](images/AssistiveLabels/HelperText.jpg)
+![helper label](images/AssistiveLabels/maui-textinputlayout-helpertext.jpg)
```
Comment (PrithisIyyappan) [inline MAUI/TextInputLayout/Assistive-Labels.md:43]: Please check and update.

File: MAUI/ComboBox/Selection.md [hunk 2]
```diff
@@ -163,7 +163,7 @@ When the [TokensWrapMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.In
 
 {% endtabs %}
 
-![.NET MAUI ComboBox Wrap mode.](Images/Selection/net-maui-combobox-wrapmode.png)
+![.NET MAUI ComboBox Wrap mode with drop down list.](Images/Selection/net-maui-combobox-wrapmode-dropdownlist.png)
```
Comment (PrithisIyyappan) [inline MAUI/ComboBox/Selection.md:166]: revert it.

File: MAUI/ComboBox/Selection.md [hunk 3]
```diff
@@ -133,7 +133,7 @@ When setting [MultiSelectionDisplayMode](https://help.syncfusion.com/cr/maui/Syn
 
 {% endtabs %}
 
-![.NET MAUI ComboBox with MultiSelectionDisplayMode as Delimiter.](Images/Selection/net-maui-combobox-delimiter.jpg)
+![.NET MAUI ComboBox with MultiSelectionDisplayMode as Delimiter and Drop down List.](Images/Selection/net-maui-combobox-delimiter-dropdownlist.jpg)
```
Comment (PrithisIyyappan) [inline MAUI/ComboBox/Selection.md:136]: Revert it.

File: MAUI/ComboBox/Selection.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ The [ComboBox](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfComb
 
 ### UI Selection 
 
-The selected item can be changed interactively by selecting from the drop-down list or entering the value.
+The selected item can be changed interactively by selecting from the drop-down list or entering the value in .net maui select menu.
```
Comment (PrithisIyyappan) [inline MAUI/ComboBox/Selection.md:21]: Revert it.

File: MAUI/ComboBox/Selection.md [hunk 5]
```diff
@@ -35,7 +35,7 @@ The selected item can be changed interactively by selecting from the drop-down l
 
 The following gif image illustrates the result of the above code:
 
-![.NET MAUI ComboBox UI selection.](Images/Selection/net-maui-combobox-ui-selection.gif)
+![.NET MAUI ComboBox UI selection.](Images/Selection/net-maui-combobox-ui-selectmenu.gif)
```
Comment (PrithisIyyappan) [inline MAUI/ComboBox/Selection.md:38]: Revert it.

File: MAUI/TextInputLayout/images/AssistiveLabels/Helpertext.jpg
Comment (PrithisIyyappan) [inline MAUI/TextInputLayout/images/AssistiveLabels/Helpertext.jpg:1]: Change back to HelperText.jpg not Helpertext.jpg

File: MAUI/TextInputLayout/Assistive-Labels.md [hunk 6]
```diff
@@ -73,7 +73,7 @@ inputLayout.Content = new Entry();
 
 {% endtabs %}
 
-![error label](images/AssistiveLabels/ErrorText.jpg)
+![.NET MAUI error label](images/AssistiveLabels/ErrorText.jpg)
```
Comment (PrithisIyyappan) [inline MAUI/TextInputLayout/Assistive-Labels.md:76]: Change to maui-textinputlayout-error label

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**